sql-to-dbsp-compiler/lib/sqllib/src/interval.rs

```diff
@@ -66,6 +67,53 @@ where
     }
 }
 
+impl SerializeWithContext<SqlSerdeConfig> for ShortInterval {
```

Comment:
A simpler way to implement this is to annotate the types with [`#serde[transparent]` ](https://serde.rs/container-attrs.html#transparent) and auto-devive `Serialize/Deserialize` impls and then use the `(de)serialize_without_context` macro to derive `(De)SerializeWithContext`